### ISSUE TITLE
`Bug`: Unavailable option `-watchdog` in `startvm`

### ISSUE THREAD

**Issue by gyptazy** [gardenlinux/gardenlinux#1854]
**What happened:**
Using QEMU on newer systems (even Debian Bookworm) will report, that the watchdog option is not more present. Therefore, `startvm` will fail:

```
Press any key to continue...
qemu-system-x86_64: -watchdog: invalid option
```

**Comment by fwilhe** [comment]
Not sure about amd64, but at least for aarch64 this was fixed in https://github.com/gardenlinux/gardenlinux/pull/1777

**Comment by gyptazy** [comment]
> Not sure about amd64, but at least for aarch64 this was fixed in #1777

Unfortunately, this also applies to some newer versions of `qemu-system-x86_64`. The current implementation was therefore also only for Debian based systems before bookworm. IMHO, I think we can safely remove this. You might have a look at my PR #1948. Let me know if you like to have any changes.

Have a great weekend!

Cheers,
gyptazy
